app/models/category_transaction.rb

```diff
@@ -0,0 +1,3 @@
+class CategoryTransaction < ApplicationRecord
+  belongs_to :category
+end
```

Comment:
- [ ] There should be an association between `transactions` and `category_transaction`. Here it should be `belongs_to :transaction`.